<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtomsWindow</class>
 <widget class="QDialog" name="AtomsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FerromagnetWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background: rgba(214, 219, 233,255);
}

QComboBox{
background: rgba(214, 219, 233,255);
border: none;
}

QComboBox:hover{
background: rgba(255, 255, 191,255);
border:  2px solid rgba(255, 170, 0,255);
}

QPushButton:hover{
background: rgba(255, 255, 191,255);
border:  2px solid rgba(255, 170, 0,255);
}

QPushButton{
border: none;
background: rgba(214, 219, 233,255);
}

QTableWidget {
    background: rgba(214, 219, 233,255);
border: none;
}

QHeaderView::section{
background-color: rgba(235, 241, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1,1">
     <item>
      <widget class="QCheckBox" name="checkAtoms">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>show atoms as speres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMapValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCMaps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetGeometry">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2,1">
       <item>
        <widget class="QLabel" name="labelCMin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>minimum color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCMin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>cMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCMax">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>maximum color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCMax">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>cMax</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
       <item>
        <widget class="QLabel" name="labelCutoffs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>set cutoffs for colormap.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomTableWidget" name="tableWidgetCutoff">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout2" stretch="3,2">
       <item>
        <widget class="QLabel" name="labelMTip">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>MTip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomTableWidget" name="tableWidgetMtip">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>CustomTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>readparams()</slot>
 </slots>
</ui>
